api/cli.py

```
@@ -62,6 +62,18 @@ def _check_connection(host: str, port: int) -> bool:
 def ensure_db() -> None:
     """Ensure FalkorDB is running, auto-starting a Docker container if needed."""
 
+    from .db import create_falkordb, is_lite_backend
+
+    if is_lite_backend():
+        try:
+            db = create_falkordb()
+            db.connection.ping()
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)